Mathlib/Topology/MetricSpace/MetricSeparated.lean

```diff
@@ -1,38 +1,74 @@
 /-
-Copyright (c) 2021 Yury Kudryashov. All rights reserved.
+Copyright (c) 2021 Yury Kudryashov, Yaël Dillies. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Yury Kudryashov
+Authors: Yury Kudryashov, Yaël Dillies
 -/
 import Mathlib.Topology.EMetricSpace.Defs
 
 /-!
-# Metric separated pairs of sets
+# Metric separation
 
-In this file we define the predicate `Metric.AreSeparated`. We say that two sets in an (extended)
-metric space are *metric separated* if the (extended) distance between `x ∈ s` and `y ∈ t` is
-bounded from below by a positive constant.
+This file defines a few notions of separations of sets in a metric space.
```

Comment:
Let's make it even clearer here that the two notions are genuinely different from each other: one is about points in a set being separated and the other is about sets themselves being separated

Reply:
nice!